### ISSUE TITLE
Trouble with irl use of train.py script

### ISSUE THREAD

**Issue by KrotAphanasiy** [arunponnusamy/gender-detection-keras#25]
Hey, you should really work on learning parametres, coz what I got using your default train.py was just kinda very unstable validation loss (jumping literally from 0.01 to 2.0+) during the whole learning process. It makes model unable to be used in real situations.. 
![p_TZsxWC6IE](https://user-images.githubusercontent.com/52762504/87722435-c1f61680-c7c0-11ea-9fce-1f9c6d74a476.jpg)
![Снимок экрана от 2020-07-17 00-02-23](https://user-images.githubusercontent.com/52762504/87722470-d2a68c80-c7c0-11ea-932b-7e4ad27724af.png)

First is your variant, but isn`t my result better? Or you just threw that script like a template?

**Comment by tayyabnasem** [comment]
> Hey, you should really work on learning parametres, coz what I got using your default train.py was just kinda very unstable validation loss (jumping literally from 0.01 to 2.0+) during the whole learning process. It makes model unable to be used in real situations..
> ![p_TZsxWC6IE](https://user-images.githubusercontent.com/52762504/87722435-c1f61680-c7c0-11ea-9fce-1f9c6d74a476.jpg)
> ![Снимок экрана от 2020-07-17 00-02-23](https://user-images.githubusercontent.com/52762504/87722470-d2a68c80-c7c0-11ea-932b-7e4ad27724af.png)
> 
> First is your variant, but isn`t my result better? Or you just threw that script like a template?

Your approach is better Can you share the model you used while obtaining these results
